internal/pwru/output.go

```diff
@@ -131,6 +132,16 @@ func (o *output) Print(event *Event) {
 		funcName = fmt.Sprintf("0x%x", addr)
 	}
 
+	if strings.HasPrefix(funcName, "bpf_prog_") && strings.HasSuffix(funcName, "[bpf]") {
+		// The name of bpf prog is "bpf_prog_<id>_<name>  [bpf]". We want to
+		// print only the name.
+		items := strings.Split(funcName, "_")
```

Comment:
Not sure about if we shoud strip prefix and suffix for bpf prog. When `--filter-all-kmods`, the module names aren't stripped. cc @brb

Reply:
Hmm, maybe having the suffix (`[bpf]`) might confuse users that a traced function is coming from a kmod?